xwiki-platform-core/xwiki-platform-flamingo/xwiki-platform-flamingo-skin/xwiki-platform-flamingo-skin-resources/src/main/resources/flamingo/less/misc.less

```diff
@@ -216,6 +216,7 @@ div.suggestItems .hide-button {
     margin-top: .3em;
   }
   .submitSection {
+    margin-top: floor(@font-size-base * 1.4); // Same as `.xform dl dt`
```

Comment:
Minor comment, I think we normally put comments on their own line in our code style.

Reply:
Addressed in https://github.com/xwiki/xwiki-platform/pull/3379/commits/636b1f96a74551b0b8bb53f91481d5f9ec9f6074 :+1: